packages/frontend/src/hooks/performBuyOrSellUSN.js

```diff
@@ -55,13 +55,13 @@ export const usePerformBuyOrSellUSN = () => {
         const account = await wallet.getAccount(accountId);
         const usnContract = new nearApiJs.Contract(
             account,
-            USN_CONTRACT,
+            CONFIG,
```

Comment:
Shouldn't there be a string? But object is passed

Reply:
Seems I didn't delete this file. As I can see we're not using it anywhere, so let's delete it.